test/src/eks_blueprints_e2e_test.go

```diff
@@ -42,27 +42,22 @@ var (
 
 	destroyModules = []string{
 		"module.eks_blueprints_kubernetes_addons",
-		"module.eks_blueprints",
-		"module.vpc",
+		"module.eks",
 		"full_destroy",
 	}
 
-	/*Update the expected Output variables and values*/
-	outputParameters = [...]Outputs{
-		{"vpc_cidr", "10.0.0.0/16", "equal"},
-		{"vpc_private_subnet_cidr", "[10.0.10.0/24 10.0.11.0/24 10.0.12.0/24]", "equal"},
-		{"vpc_public_subnet_cidr", "[10.0.0.0/24 10.0.1.0/24 10.0.2.0/24]", "equal"},
-		{"eks_cluster_id", "aws-terra-test-eks", "equal"},
-		{"eks_managed_nodegroup_status", "[ACTIVE]", "equal"},
-	}
+	// /*Update the expected Output variables and values*/
+	// outputParameters = [...]Outputs{
+	// 	{"eks_managed_nodegroup_status", "[ACTIVE]", "equal"},
```

Comment:
This is another test checking Terraform outputs - we don't need to test the cluster in these examples